quick_purchase/models/product_product.py

```diff
@@ -0,0 +1,96 @@
+# coding: utf-8
+# © 2014 Today Akretion
+# @author Sébastien BEAU <sebastien.beau@akretion.com>
+# @author Mourad EL HADJ MIMOUNE <mourad.elhadj.mimoune@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+
+from openerp import api, fields, models
+
+
+class ProductProduct(models.Model):
+    _inherit = 'product.product'
+
+    qty_to_purchase = fields.Float(
+        compute='_compute_purchase_qty',
+        inverse='_set_purchase_qty',
+        oldname='purchase_qty',
+    )
+    po_line_ids = fields.One2many(
+        'purchase.order.line', 'product_id',
+        help='Technical: used to compute quantities.')
+    stock_quant_ids = fields.One2many(
+        'stock.quant', 'product_id',
+        help='Technical: used to compute quantities.')
+    stock_move_ids = fields.One2many(
+        'stock.move', 'product_id',
+        help='Technical: used to compute quantities.')
+
```

Comment:
Nowhere now. I used it first to compute real_incomming_qty. But this fields in not necessary in the new warehouse models of v 8.0. I'ill remove it.